### DISCUSSION TITLE
Enlarge the Channel Scrollbar for Easier Access & Function

### DISCUSSION THREAD

**Discussion by JazzyJeff123** [mattermost/community#8] [Enhancements]
When using a laptop touchpad it's somewhat difficult to point and click on the channel scrollbar. We could have it appear with a larger overall display for easier access. This is especially helpful when a user is a member of many channels and their scrollbar gets smaller as their channel number increases.

**Comment by esethna** [comment]
Totally agree on touch devices, and then on non-touch devices we should improve the scroll bar UI to make it even LESS intrusive and consistently styled across the different panes (sidebar, center channel, RHS, etc)

**Comment by matthewbirtch** [comment]
Can't you gesture swipe up/down to scroll on touch devices though?

**Reply by Willyfrog** [reply]
I've seen this requested a number of times. While some might scroll using their hands I think the user has either a mouse or a pen connected and they try to keep using that as moving the hand around to scroll then back to your device is a not a comfortable movement

**Reply by esethna** [reply]
Also a lot of users we know about use stylus' and like to grab the scroll bar to scroll. Although we should understand the use cases more and why scrolling without grabbing the bar doesn't work

**Comment by esethna** [comment]
@JazzyJeff123 can you help us understand why you use the scrollbar to scroll the sidebar rather than a swiping motion over the sidebar? Just trying to understand the use case so we can optimize the UX

**Reply by JazzyJeff123** [reply]
I actually use Mattermost online and not on a mobile device. I feel that the scrollbar is too small and needs to be thicker when visible, I also don't like that it becomes smaller as my channel count increases. The larger scrollbar picture is from my Contributors account that has few channels. The small scrollbar is from my staff account.
![image](https://user-images.githubusercontent.com/86306765/123169495-5ba15b80-d447-11eb-940f-a45793a28e7d.png)
![image](https://user-images.githubusercontent.com/86306765/123169585-796ec080-d447-11eb-97ff-47bb780d5d3f.png)

**Reply by matthewbirtch** [reply]
> I actually use Mattermost online and not on a mobile device. I feel that the scrollbar is too small and needs to be thicker when visible, I also don't like that it becomes smaller as my channel count increases. The larger scrollbar picture is from my Contributors account that has few channels. The small scrollbar is from my staff account.

So, it sounds like you don't use a mouse with a scroll wheel on it?

**Reply by JazzyJeff123** [reply]
I use my laptop touchpad 90% of the time which makes hovering over and clicking on the smaller scrollbar difficult

**Reply by esethna** [reply]
@JazzyJeff123 scrolling by swiping on the touchpad with two fingers doesn't work for you? Or is there a reason you don't do that?

**Reply by JazzyJeff123** [reply]
@esethna I don't have the scrolling by swiping feature on my Lenovo. The suggestion to have a larger more pronounced scrollbar is just that, maybe I have a minority opinion where the majority of our customer base doesn't need a larger scrollbar. I just wanted to point it out so that way you know it's a functional pain point for someone and that way if you ever hear the same thing from a customer review of our product there might be more of a case to enlarge it.

**Reply by esethna** [reply]
@JazzyJeff123 totally, really appreciate your feedback, just wanted to understand it more. That makes a lot of sense if you don't have a swipe to scroll feature!
